engine-core/src/main/java/io/nosqlbench/engine/core/logging/LoggerConfig.java

```diff
@@ -116,9 +115,20 @@ Configuration createConfiguration(final String name, ConfigurationBuilder<BuiltC
 
         Level internalLoggingStatusThreshold = Level.ERROR;
         Level builderThresholdLevel = Level.INFO;
-//        Level rootLoggingLevel = Level.INFO;
 
-        RootLoggerComponentBuilder rootBuilder = builder.newRootLogger(ROOT_LOG_LEVEL);
+        Level fileLevel = Level.valueOf(getEffectiveFileLevel().toString());
+        Level consoleLevel = Level.valueOf(this.consoleLevel.toString());
+
+        // configure the ROOT logger the same way as the File level
+        // this is because the fileLevel is supposed to show more than the console
+
+        // therefore, it is very important that the ROOT level is as much specific as possible
+        // because NB code and especially third party libraries may rely on logging guards (if logger.isDebugEnabled())
+        // to reduce memory allocations and resource waste due to debug/trace logging
+        // if you set ROOT to ALL or to TRACE then you will trigger the execution of trace/debugging code
+        // that will affect performances and impact on the measurements made with NB
+        Level rootLoggingLevel = fileLevel;
```

Comment:
wouldn't `rootLoggingLevel` have to be either `fileLevel` or `consoleLevel`, picking the finer grain one?

Reply:
Oh sorry, I didn't notice your comment "We already have a getEffectiveFileLevel() that computes the most verbose (specific in log4j2 terms) logger level."

Reply:
This is fine.

Reply:
it is already computed...see here:
```
private NBLogLevel getEffectiveFileLevel() {
        if (fileLevel.isGreaterOrEqualTo(consoleLevel)) {
            return fileLevel;
        } else {
            return consoleLevel;
        }
    }
```